### PR TITLE
Raj_Assignment_21_40min_Done

### PR DESCRIPTION
**Pull request by RajAJun20**
Assignment_21_P1
Assignment_21_P2

### CODE REVIEW

**Comment by KrishnaKTechnocredits** [comment]
Your reviewer is Anshu, why you have not requested her to review your code.

File: src/rajAJun20/FibonacciNumbers.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+public class FibonacciNumbers {
+	
+	void findFibonacciNumbers(int input) {
+			int sum = 0;
+			int firstNum = 0  , nextNum = 1 ;
+			System.out.println("Fibonacci series for input " +input+ " is :");
+			for(int index = 0 ; index < input ; index++ ) {
+				sum = firstNum + nextNum;
+				System.out.println(nextNum);
```
Comment (AnshuKJun20) [inline src/rajAJun20/FibonacciNumbers.java:11]: Here you have to write System.out.print(nextNum); for printing in same line. Println is used  for printing in nextline. This is also correct but we have try to print exactly same output given in question.

**Review by AnshuKJun20 — APPROVED** [review]
Once you have to  look on my comments. Otherwise all are well written and i am going to approve it.

**Comment by RajAJun20** [comment]
Ok Anshu I got it.

**Pull request merged**